src/reducers/words.js

```diff
@@ -0,0 +1,25 @@
+import dictionary from '../fixtures/words.json';
+import { UPDATE_SEARCH_TERM, UPDATE_COLOR } from '../actions/words';
+
+const initialState = {
+  dictionary,
+  color: '',
```

Comment:
```suggestion
  color: '#000000',
```
Recommend defaulting to a valid rgb value to prevent browser warnings.